jetty-server/src/main/java/org/eclipse/jetty/server/Connector.java

```diff
@@ -97,4 +98,21 @@ public interface Connector extends LifeCycle, Container, Graceful
      * @return The connector name or null.
      */
     public String getName();
+
+    public interface VirtualThreadsConfigurable
+    {
+        public default boolean isInvokeApplicationWithVirtualThreads()
```

Comment:
This is not the right name.
Nor am I convinced if it should be a connection configuration.